### ISSUE TITLE
Visualize a taxonomic tree with sigma. Is it possible?

### ISSUE THREAD

**Issue by majuss** [jacomyal/sigma.js#861]
I want to visualize a dataset of 90.000 species and their taxonomic path with sigma.js.
Will it be possible to render a static tree with these large amount of nodes or will the browser just hang?

I looked at examples like: [This](http://www.kenedict.com/netwerc/#) which already hangs completely at 7000 nodes :(
Does anybody have informations about the scalability or should I use [SNAP](http://snap.stanford.edu/data/other.html?)

**Comment by Yomguithereal** [comment]
Hum... 90k nodes and I guess you have as much edges also ~90k since we are speaking of a tree. It starts to be a little too much for a web browser to render. I know I already rendered approx. this size of graph but it will be laggy on most computers and you'll have to sacrifice some visual features. Does your viz need to be dynamic? Else I would recommend to develop a custom renderer using webgl.

**Comment by majuss** [comment]
Thanks for the answer! 
No I don't need dynamics or anything, just a static .svg-like graph with 2 gradient colors. In sigma you can disable all the visual candies?

**Comment by Yomguithereal** [comment]
Yes you can disable most of it. I must tell you however that sigma has a slight synchronous performance overhead when reading your graph for the first time and this might be a problem for you. That's why we are currently working on [graphology](https://graphology.github.io/) and a new version of sigma.
